### DISCUSSION TITLE
Welcome to vehicle-counting-yolov5 Discussions!

### DISCUSSION THREAD

**Discussion by mahimairaja** [mahimairaja/vechicle-counting-yolo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vikrampalan** [comment]
Hey,  could you help me regarding how to use this
I'm new to world of coding so need your help to understand these things.

**Reply by mahimairaja** [reply]
Kindly follow the steps as listed. 
If you have any doubt do drop a comment below.
